classes/autotools.oeclass

```diff
@@ -48,7 +48,16 @@ autotools_configure[emit] = "do_configure"
 do_configure () {
     do_configure_autotools
 }
+AUTOCONFDIR = ""
+UPDATE_GNUCONF ?= "0"
+UPDATE_GNUCONF:TARGET_CPU_aarch64 = "1"
+
 do_configure_autotools () {
+    if [ -a "${UPDATE_GNUCONF}" != "0" ] ; then
+        rm -f ${S}${AUTOCONFDIR}/config.{guess,sub}
+        cp ${TOPDIR}/meta/core/scripts/config.{guess,sub} ${S}${AUTOCONFDIR}
```

Comment:
Oh, there is a problem with this approach.  By taking files out from manifest like this, you are bypassing the signature system.  So changes to config.{guess,sub} is not tracked at all, which is clearly not a good thing.

To fix this, we probably need to create a recipe for the config.{guess,sub} files, and then add a dependency to this in CLASS_DEPENDS if UPDATE_GNUCONF is set, and then copy from stage instead.